<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="kavinslaptop" failures="0" tests="2" name="Class.testn" time="0.017" errors="1" timestamp="2022-03-19T17:27:46 EDT" skipped="0">
  <testcase classname="Class.testn" name="writetestcase" time="0.002"/>
  <system-out/>
  <testcase classname="Class.testn" name="executetestcase" time="0.015">
    <error message="Unresolved compilation problems: 
	Syntax error on token &quot;void&quot;, record expected
	void is an invalid type for the variable defectlog
	Syntax error, insert &quot;Identifier (&quot; to complete MethodHeaderName
	Syntax error, insert &quot;)&quot; to complete MethodDeclaration
	Syntax error, insert &quot;;&quot; to complete MethodDeclaration
	Syntax error, insert &quot;}&quot; to complete RecordBody
	Syntax error, insert &quot;)&quot; to complete Expression
	Syntax error, insert &quot;AssignmentOperator Expression&quot; to complete Assignment
	Syntax error, insert &quot;;&quot; to complete Statement
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
Syntax error on token "void", record expected
void is an invalid type for the variable defectlog
Syntax error, insert "Identifier (" to complete MethodHeaderName
Syntax error, insert ")" to complete MethodDeclaration
Syntax error, insert ";" to complete MethodDeclaration
Syntax error, insert "}" to complete RecordBody
Syntax error, insert ")" to complete Expression
Syntax error, insert "AssignmentOperator Expression" to complete Assignment
Syntax error, insert ";" to complete Statement

at Class.testn.executetestcase(testn.java:16)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- executetestcase -->
  <system-out/>
</testsuite> <!-- Class.testn -->
